specification/metrics/billedunitprice.md

```diff
@@ -0,0 +1,31 @@
+# Billed Unit Price
+
+The Billed Unit Price represents the unit price for a single pricing measurement unit, which incorporates various discounts like volume/tier-based rates, negotiated discounts and commitment-based discounts. Billed Unit Price does not amortize upfront charges (one-time or recurring). This unit price can match the List Unit Price, if there are no discounts in place. This pricing is denominated in the  [Billing Currency](#billingcurrency). It is used to calculate Billed Cost and savings based on financial optimization activities.
```

Comment:
Should we mention that this is a generated unit price added for convenience - since there's no published price like this and the providers outside of AWS may not even have this consistently. 

Should we also say billed unit prices may change over time as the benefits of negotiated discounts and commitment based discounts application priority changes to make it clear why this rate may jump around drastically (e.g. might be $0.00 one hour and be something much more expensive in another hour if someone else got the RI)? 

Thinking about if we would avoid adding this similar to effective unit price rate. Both seem to be cake not ingredient. 

Also, should we avoid implying this is used to calculate Billed Cost (and instead say the other way around if we're keeping this).

(As I make these suggestions, I fully recognize I wrote some of this.. :( so, sorry for not bringing this up earlier)

Reply:
Worth a discussion!